README.md

```diff
@@ -112,7 +127,7 @@ flow-merge --help
 You can display the config yaml schema and the default values by running:
 
 ```bash
-flow-merge schema
+flow-merge schema | jq
```

Comment:
Hmm, jq might not be installed, as it's not in the standard environment.
Maybe we need to just suggest it.
There are also some more interactive and beginner friendly options than jq.

Reply:
Ok great